### PR TITLE
[Docs] fix OR to ORFS build links, readme

### PR DESCRIPTION
**Pull request by luarss**
@vvbandeira @vijayank88

### CODE REVIEW

File: docs/user/Build.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 ## Install dependencies
 
 To install dependencies to develop or build and run OpenROAD locally,
-follow [this procedure](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html).
+follow [this procedure](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html#clone-and-install-dependencies).
```
Comment (vvbandeira) [inline docs/user/Build.md:6]: We should re-write this with the correct instructions on how to run the installer.

Comment (luarss) [reply docs/user/Build.md]: @vvbandeira I just added some instructions, let me know if that works

File: README.md [hunk 2]
```diff
@@ -50,12 +50,12 @@ includes GlobalFoundries shuttles, design contests and IC design
 workshops. The OpenROAD flow has been successfully used to date
 in over 600 silicon-ready tapeouts for technologies up to 12nm.
 
-## OpenROAD Flow Stages
+## Getting Started with OpenROAD Flow-Scripts
```
Comment (vijayank88) [inline README.md:53]: Getting Started with OpenROAD-flow-scripts

**Review by vijayank88 — CHANGES_REQUESTED** [review]

**Review by vvbandeira — APPROVED** [review]
@dralabeing this looks good to me

**Comment by luarss** [comment]
@vvbandeira @vijayank88 @dralabeing Any updates on this PR?

File: docs/index.md [hunk 3]
```diff
@@ -37,28 +37,16 @@ The paper is also available from [ACM Digital Library](https://dl.acm.org/doi/10
...
 ## Documentation
 
 The OpenROAD Project has two releases:
 
-### [Application](https://github.com/The-OpenROAD-Project/OpenROAD)
-
-The application is a standalone binary capable of performing RTL-to-GDSII SoC design,
-from logic synthesis and floorplanning through detailed routing with metal fill insertion,
-signoff parasitic extraction and timing analysis.
+- Application ([github](https://github.com/The-OpenROAD-Project/OpenROAD)) ([docs](main/README.md)): The application is a standalone binary capable of performing RTL-to-GDSII SoC design.
```
Comment (dralabeing) [inline docs/index.md:44]: Change description to : the application is a standalone binary for digital place and route that can be used by any other RTL-GDSII flow controller.

File: docs/index.md [hunk 4]
```diff
@@ -37,28 +37,16 @@ The paper is also available from [ACM Digital Library](https://dl.acm.org/doi/10
...
+- Flow ([github](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts)) ([docs](https://openroad-flow-scripts.readthedocs.io/en/latest/)): The flow is a set of integrated scripts that allow for RTL-to-GDSII flow using open-source tools.
```
Comment (dralabeing) [inline docs/index.md:45]: Change description to: This is the native OpenROAD flow that consists of a set of  integrated scripts for an autonomous RTL-GDSII flow using OpenROAD and other open-source tools.

**Review by dralabeing — APPROVED** [review]

**Review by vijayank88 — APPROVED** [review]

**Pull request merged**